typescript/karma.conf.js

```diff
@@ -29,8 +29,10 @@ module.exports = function (config) {
       subdir: '.',
       reporters: [
         { type: 'html' },
-        { type: 'text-summary' }
-      ]
+        { type: 'cobertura' },
+        { type: 'text-summary' },
+        { type: 'lcovonly' },
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/newsbridge/sandbox/pullRequest?prid=7539937)
```suggestion
        { type: "lcovonly" },
```